tests/integration/config/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Not sure if we'd want this to be committed as this is a real login isn't it?

Reply:
will this now in the git history?

Reply:
Not sure. @LGin-BJSS - I am not aware of a way to delete a specific commit on GH, are you?